feat: update translation docs to be part of `contributing` (#1884)
medic/cht-docs: content/en/community/contributing/translations.md

---
title: Translations
linkTitle: Translations
weight: 4
description: >

>   Internationalizing CHT Core.

Reviewer: ```suggestion
  Internationalizing CHT Core
```

relatedContent: >
  building/translations/overview
aliases:
  - /core/overview/translations
  - /building/translations/managing-translations
  - /building/translations/managing
---

> 

Reviewer: ```suggestion
{{< hextra/hero-subtitle >}}
  Internationalizing CHT Core
{{< /hextra/hero-subtitle >}}
```

> [!TIP]
> Open [an issue](https://github.com/medic/cht-core/issues/new) if you are interested in translating the CHT into a different language, and make it available to the community.

CHT Core currently has built-in support for the following languages:

| Language   | Language Code | Status     |
|------------|---------------|------------|
| Arabic     | `ar`          | Incomplete |
| Bambara    | `bm`          | Incomplete |
| English    | `en`          | Complete   |
| Spanish    | `es`          | Complete   |
| French     | `fr`          | Complete   |
| Hindi      | `hi`          | Incomplete |
| Indonesian | `id`          | Incomplete |
| Nepali     | `ne`          | Complete   |
| Swahili    | `sw`          | Complete   |

Languages marked as "Complete" are fully translated and any new user-facing text will always be translated into these languages. Languages marked as "Incomplete" may be missing translations for some keys and/or the CHT community needs additional volunteers to help support this language by translating new keys. 

> If you are interested in helping to support any of these languages, please come introduce yourself in the [Translation category](https://forum.communityhealthtoolkit.org/c/product/translations/35) on the CHT Community Forum! (No technical knowledge/experience is needed to help with translations.)

Reviewer: ❤️

See the [localization documentation]({{< ref "building/translations/overview" >}}) for more information on how to manage custom translations for a particular CHT instance.

## Overview

> Translation files live in the GitHub repo. These translation files are [properties files](https://en.wikipedia.org/wiki/.properties), which are a series of keys and their corresponding values. The English file is used by default, and as such, it contains the entire set of keys. If any key is missing from another language file the English value is used.

Reviewer: ```suggestion
Translation files live in the cht-core GitHub repository. These translation files are [properties files](https://en.wikipedia.org/wiki/.properties), which are a series of key-value pairs. The English file is used by default, and as such, it contains the entire set of keys. If any key is missing from another language file, the English value is used.
```

Reviewer: nitpicks
